misc/toml-parser/src/test/resources/semantic/existing-node.toml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+key = "first"
+key = "second"
+foo.mow = "test"
+
+[foo]
```

Comment:
Shouldn't we also say existing node/table `foo` here.

Reply:
I'm not clear on this. 
```toml
foo.mow = "test"

[foo]
mow = "wew"
```
in the following example, line `mow = "wew"` is the duplicate entry,

if you change the file to
```toml
foo.mow = "test"

[foo]
bar = "wew"
```
this is a valid toml syntax.

In Toml AST this gets represented as.
[foo]
mow = "test"
bar = "wew"

Reply:
Discussed offline and Fixed.